.github/workflows/test-on-projects.yml

```diff
@@ -0,0 +1,63 @@
+name: test on projects
+
+on:
+  push:
+    branches:
+      - main
+      - open_repo
+  workflow_dispatch: {}
```

Comment:
Please, delete open_repo from this. It is a temporary branch
Instead of it you can run these tests on PR in the main branch